egs2/librimix/enh_diar1/conf/tuning/train_diar_enh_convtasnet_adapt.yaml

```diff
@@ -0,0 +1,99 @@
+optim: adam
+init: xavier_uniform
+max_epoch: 50
+batch_type: folded
+batch_size: 4
+accum_grad: 4
+iterator_type: chunk
+chunk_length: 24000
+num_workers: 4
+optim_conf:
+    lr: 1.0e-03
+    eps: 1.0e-07
+    weight_decay: 0
+patience: 4
+val_scheduler_criterion:
+- valid
+- loss
+best_model_criterion:
+-   - valid
+    - loss_enh
+    - min
+keep_nbest_models: 1
+scheduler: reducelronplateau
+scheduler_conf:
+    mode: min
+    factor: 0.5
+    patience: 1
+
+diar_input_size: 128 # same as bottleneck_dim
+subtask_series: ["enh", "diar"]
+
+enh_model_conf:
+    loss_type: si_snr
+diar_model_conf:
+    diar_weight: 0.2
+    attractor_weight: 0.2
+enh_encoder: conv
+enh_encoder_conf:
+    channel: 512
+    kernel_size: 16
+    stride: 8
+enh_decoder: conv
+enh_decoder_conf:
+    channel: 512
+    kernel_size: 16
+    stride: 8
+enh_separator: tcn_nomask
+enh_separator_conf:
+    layer: 8
+    stack: 3
+    bottleneck_dim: 128
+    hidden_dim: 512
+    kernel: 3
+    causal: False
+    norm_type: "gLN"
+enh_mask_module: multi_mask
+enh_mask_module_conf:
+    max_num_spk: 3
+    mask_nonlinear: relu
+    bottleneck_dim: 128
+label_aggregator: label_aggregator
+label_aggregator_conf:
+    win_length: 256
+    hop_length: 64  # This value should be set to:
+                    # enh_encoder stride * subsampling factor (corresponding to diar_encoder input_layer)
+                    # subsampling factors: 1 (linear), 2 (conv2d2), 4 (conv2d), 6 (conv2d6), 8 (conv2d8)
+diar_frontend: null
+frontend: null # this is only required when "diar_front_end: null"
+# uncomment the following lines to use concatenated log-mel filterbank and bottleneck features
+# diar_frontend: default
```

Comment:
Can you clean the config?

Reply:
Should I split it into 2 files? I could otherwise write some instructions in README.

Reply:
Maybe spliting it into 2 files is better.